### PR TITLE
Dashboard section of osra

### PR DESCRIPTION
**Pull request by achock**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,14 @@
 Osra::Application.routes.draw do
 
+  namespace :hq do
+  get 'dashboard/index'
+  end
```
Comment (razorcd) [inline config/routes.rb]: why is this needed since we have line 11?

File: spec/views/hq/dashboard/index.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "hq/dashboard/index.html.erb", type: :view do
```
Comment (gmunumel) [inline spec/views/hq/dashboard/index.html.erb_spec.rb]: Be careful with the indentation. Try to use a 2 or 4 space indent.

File: app/controllers/hq_controller.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 class HqController < ApplicationController
   NAVIGATION_BUTTONS= [
     { text: 'OSRA', href: 'root_path', path_regex: /^\/$/, glyph: 'glyphicon-home' },
-    { text: 'Dashboard', href: '"/404.html"', path_regex: /^\/hq(\/)?$/, glyph: 'glyphicon-dashboard' },
+    { text: 'Dashboard', href: '"dashboard"', path_regex: /^\/hq\/dashboard/, glyph: 'glyphicon-dashboard' },
```
Comment (gmunumel) [inline app/controllers/hq_controller.rb]: You can change `"dashboard"` with  `hq_dashboard_path` since you created the route for it. If you are not sure about the route please take a look at `rake routes`.

File: app/assets/javascripts/hq/dashboard.coffee [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (gmunumel) [inline app/assets/javascripts/hq/dashboard.coffee]: I don't see the point of adding a `.coffee` file that we are not using it at the moment. My opinion is to added only when  we *really* need it.

File: app/assets/stylesheets/hq/dashboard.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the hq/dashboard controller here.
```
Comment (gmunumel) [inline app/assets/stylesheets/hq/dashboard.scss]: Same opinion as the above.

File: spec/controllers/hq/dashboard_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Hq::DashboardController, type: :controller do
+
+  describe "GET #index view" do
+  	before(:each) do
+    	sign_in instance_double(AdminUser)
```
Comment (gmunumel) [inline spec/controllers/hq/dashboard_controller_spec.rb]: Please be careful with the indentation.

File: app/helpers/hq/dashboard_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module Hq::DashboardHelper
+end
```
Comment (gmunumel) [inline app/helpers/hq/dashboard_helper.rb]: Is this *really* need it?

Comment (achock) [inline app/helpers/hq/dashboard_helper.rb]: Most of these are generated by the generator. So this is why they are there.

Comment (gmunumel) [inline app/helpers/hq/dashboard_helper.rb]: Ok. So I guess we can get rid of them and add them only in case we need them.
